idp/src/main/java/dk/itst/oiosaml/idp/controller/saml/SingleSignOnController.java

```diff
@@ -14,7 +14,7 @@
 import org.opensaml.saml.common.messaging.context.SAMLEndpointContext;
 import org.opensaml.saml.common.messaging.context.SAMLPeerEntityContext;
 import org.opensaml.saml.common.xml.SAMLConstants;
-import org.opensaml.saml.saml2.binding.encoding.impl.HTTPPostEncoder;
+import org.opensaml.saml.saml2.binding.encoding.impl.HTTPPostSimpleSignEncoder;
```

Comment:
Didn't we reach the conclusion that HTTPPostSimpleSignEncoder was wrong? We don't want to double sign the content.

Reply:
done